numpy/core/code_generators/generate_umath.py

```diff
@@ -814,13 +814,13 @@ def english_upper(s):
     Ufunc(1, 1, None,
           docstrings.get('numpy.core.umath.isinf'),
           None,
-          TD(inexact, out='?'),
+          TD(noint, out='?'),
           ),
 'isfinite':
     Ufunc(1, 1, None,
           docstrings.get('numpy.core.umath.isfinite'),
           None,
-          TD(inexact, out='?'),
+          TD(noint, out='?'),
```

Comment:
What happens if I have multiple overlapping `TD` calls? e.g., if I hadn't deleted the line `TD(inexact, out='?')` here.